_topic_maps/_topic_map.yml

```diff
@@ -430,34 +430,38 @@ Topics:
   Topics:
   - Name: Preparing to install on bare metal
     File: preparing-to-install-on-bare-metal
-  - Name: Installing a user-provisioned cluster on bare metal
-    File: installing-bare-metal
-  - Name: Installing a user-provisioned bare metal cluster with network customizations
-    File: installing-bare-metal-network-customizations
-  - Name: Installing a user-provisioned bare metal cluster on a restricted network
-    File: installing-restricted-networks-bare-metal
-  - Name: Scaling a user-provisioned installation with the bare metal operator
-    File: scaling-a-user-provisioned-cluster-with-the-bare-metal-operator
-  - Name: Installation configuration parameters for bare metal
-    File: installation-config-parameters-bare-metal
-- Name: Deploying installer-provisioned clusters on bare metal
-  Dir: installing_bare_metal_ipi
-  Distros: openshift-origin,openshift-enterprise
-  Topics:
-  - Name: Overview
-    File: ipi-install-overview
-  - Name: Prerequisites
-    File: ipi-install-prerequisites
-  - Name: Setting up the environment for an OpenShift installation
-    File: ipi-install-installation-workflow
-  - Name: Installing a cluster
-    File: ipi-install-installing-a-cluster
-  - Name: Troubleshooting the installation
-    File: ipi-install-troubleshooting
-  - Name: Postinstallation configuration
-    File: ipi-install-post-installation-configuration
-  - Name: Expanding the cluster
-    File: ipi-install-expanding-the-cluster
+  - Name: User-provisioned infrastructure
+    Dir: upi
+    Distros: openshift-origin,openshift-enterprise
+    Topics:
+    - Name: Installing a user-provisioned cluster on bare metal
+      File: installing-bare-metal
+    - Name: Installing a user-provisioned bare metal cluster with network customizations
+      File: installing-bare-metal-network-customizations
+    - Name: Installing a user-provisioned bare metal cluster on a restricted network
+      File: installing-restricted-networks-bare-metal
+    - Name: Scaling a user-provisioned installation with the bare metal operator
+      File: scaling-a-user-provisioned-cluster-with-the-bare-metal-operator
+    - Name: Installation configuration parameters for bare metal
+      File: installation-config-parameters-bare-metal
+  - Name: Installer-provisioned infrastructure
+    Dir: ipi
+    Distros: openshift-origin,openshift-enterprise
+    Topics:
+    - Name: Overview
+      File: ipi-install-overview
+    - Name: Prerequisites
+      File: ipi-install-prerequisites
+    - Name: Setting up the environment for an OpenShift installation
+      File: ipi-install-installation-workflow
+    - Name: Installing a cluster
```

Comment:
Topic map title consistency between the UPI title on [line 437](https://github.com/openshift/openshift-docs/pull/83982/files#diff-7c09a5ccc63f204d75ed89dd359a340a43551a4a790ae36fb9c64f6c5149211bR437) on this commit, and the IPI title on this line 457 which should instead read `Installing a installer-provisioned cluster on bare metal`.

> This would also require title change in the ADOC file itself.

Alternatively, refer to https://github.com/openshift/openshift-docs/pull/83386#issuecomment-2435695685 from @bscott-rh :
```
The decision was made during the 4.16 reorgs to remove
"with installer-provisioned infrastructure"
and
"with user-provisioned infrastructure"
from the assembly titles to improve readability.
```

This comment closes PR #83380

Reply:
I'm aware of that. He's got a PR for it. This one is really big, and I'd like to merge it and fix these other issues separately while this PR still builds, as this is modifying 201 different files. Otherwise, I'm going to get stuck in a rebase/merge conflict swamp.

Reply:
@johnwilkins Know the feeling, been stuck in the rebase/merge jail repeatedly with delays to reviewed PRs.

However please note, there is no other PR anymore, given this PR alters the structure and therefore supercedes. Please track the request and raise an additional PR after this has merged.